lms/static/js/instructor_dashboard/membership.js

```diff
@@ -454,21 +456,40 @@ such that the value can be defined later than this assignment (file load order).
                 $taskResSection.append($idsList);
                 for (j = 0, len1 = ids.length; j < len1; j++) {
                     identifier = ids[j];
-                    $idsList.append($('<li/>', {
-                        text: identifier
-                    }));
+                    if (identifier === message) {
```

Comment:
If we can do as described in comment below, that may make this block more easy to maintain and read.